src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,119 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int SIZE_INCREASE_FACTOR = 2;
+    private Node<K,V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = (Node<K,V>[]) new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int index = getValidIndex(hash(key, table.length), table.length);
+        Node<K, V> newNode = new Node<>(key, value, null);
+        Node<K, V> currentNode = table[index];
+        if (currentNode == null) {
+            table[index] = newNode;
```

Comment:
where your increment of size?

Reply:
It is here: 
        if (++size > table.length * LOAD_FACTOR) {
            resize();

Reply:
resolved